### ISSUE TITLE
Why does ConvEncoder  use 1-d kernel？

### ISSUE THREAD

**Issue by cocaer** [pravarmahajan/cnn-encoder-nmt#1]
Does ConvEncoder use 1-d kernel of (1) instead of 2-d kernel (k, d) ? In the paper(ConvSeq2Seq), W is described 2d X K X d. From my point of view， K is the width and d is the height. In fact，the convolution process is two-dimensional, but it only slides in one dimension. It’s very confusing for me that the source code uses `nn.Conv1d(num_channels_conv, num_channels_conv, kernel_size,
                                      padding=kernel_size // 2) for _ in range(num_layers)]`

**Comment by cocaer** [comment]
em....I found they are equal...thx...
